interpreter.py

```diff
@@ -2,8 +2,9 @@
 from rpython.rlib import jit
 from type_wrapper import String, Integer, Float, Ptr, List,\
                          Value, NoValue, NumericValue, BasicBlock
-from llvm_wrapper import *
+from llvm_objects import W_Module
 from state import State
+from llvm_wrapper import *
```

Comment:
Option 1:
List everything you want to import explicit.

Option 2:
do `import llvm_wrapper as llwrap`, then you can address stuff as `llwrap.something`